webapp/packages/core-localization/src/locales/en.ts

```diff
@@ -83,4 +83,7 @@ export const defaultENLocale = [
   ['app_root_session_expired_reload', 'Reload'],
   ['app_root_quota_exceeded', 'Quota exceeded'],
   ['core_eventsLog_dbeaverErrorDetails', 'Error details'],
+
+  ['ui_no_connection', "No Connections."],
```

Comment:
It's not a UI specific so better move it to the core-app package and name the token something like app_navigationTree_no_connections_placeholder